library/Versions.re

```diff
@@ -145,3 +147,20 @@ let getRemoteVersions = () => {
      )
   |> Lwt.return;
 };
+
+let isAlreadyInstalled = versionName => {
```

Comment:
I would assume it'll return a `bool`, whether the version is already installed. What do you think? Another thing we can do is "throwIfInstalled" or something like that.